app/models/ability.rb

```diff
@@ -31,6 +31,7 @@ def initialize(user) # rubocop:disable all
         can :override, :reservation_errors if AppConfig.get(:override_on_create)
         can :override, :checkout_errors if AppConfig.get(:override_at_checkout)
         can :view_detailed, EquipmentModel
+        can :hide, Announcement
```

Comment:
I noticed the redundancy but for some reason without that line checkout-person can't hide the announcement